src/diffusers/loaders.py

```diff
@@ -1073,11 +1139,24 @@ def load_lora_into_text_encoder(cls, state_dict, network_alpha, text_encoder, lo
                             f"{name}.out_proj.lora_linear_layer.down.weight"
                         ] = text_encoder_lora_state_dict.pop(f"{name}.to_out_lora.down.weight")
 
+                if state_dict_aux:
```

Comment:
```suggestion
                if state_dict_aux:
```

Think I would prefer to always pass everything through the state_dict and check via key matching:
```py
is "ff" in state_dict
```

whether we have additional layers in the state dict or not